articles/how-to-guides/create-qsharp-project.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Learn how to create a Q# project with the Quantum Development Kit (QDK)
+description: Initialize a project for quantum development with the QDK and Q# in the development environment of you choice
+author: natke
+ms.author: nakersha
+ms.date: 10/19/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.howto.createproject
+---
+
+# Create a Q# project in your development environment
+
+Learn how to create a Q# project with the QDK.
+
+A Q# project contains Q# files containing quantum code, as well as a host program to run the quantum program. You can write the host program in .NET languages such as C#, or in Python. You can also run Q# code in a Jupyter notebook using the IQ# kernel.
+
+Choose your development environment and language from the sections below:
+
+* [Python](#create-a-python-project)
+* [Jupyter notebooks](#create-a-jupyter-notebook-project)
+* [C#, using Visual Studio](#create-a-c-project-on-windows-using-visual-studio)
+* [C#, using VS Code](#create-a-c-project-using-vs-code)
+* [C#, using the command line](#create-a-c-project-using-the-dotnet-command-line-tool)
+
+## Create a Python project
+
+1. Pre-requisites
+
+     * The [Quantum Development Kit for Python](~/install-guide/index.md#develop-with-python)
+
+1. Create a folder for your project, and navigate to that folder
+
+1. Create a Q# file called `Operation.qs`, and add your Q# code to it. For example:
+
+    ```qsharp
+    namespace HelloWorld
```

Comment:
Nitpicky, sorry, but would it make sense remove the newline before the `{`? We haven't been the most consistent about that in the past, but I think there's an opportunity to be more consistent with new content.

Reply:
Done :)